server/webhook_parser.go

```diff
@@ -228,22 +228,37 @@ func appendCommentNotifications(wh *webhook) {
 
 	message := fmt.Sprintf("%s mentioned you on %s:\n>%s",
 		commentAuthor, jwh.mdKeySummaryLink(), jwh.Comment.Body)
+
 	for _, u := range parseJIRAUsernamesFromText(wh.Comment.Body) {
+		isAccountId := false
+		if strings.HasPrefix(u, "accountid:") {
```

Comment:
I agree, it always feels rickety when doing it this way.

Stripping the `accountid:` prefix is important here, because the prefix is not part of the account id itself. The stripping should actually occur in `utils.parseJIRAUsernamesFromText()`, but `appendCommentNotifications()` currently wouldn't know if it is a username or accountid if the prefix wasn't present.

If we could do something like `jwh.Issue.Fields.Assignee.Key() == u`, that would be much cleaner.

Reply:
"Stripping the accountid: prefix is important here." - right. Got it.